### PR TITLE
 Add motion correction indicies to caiman params

### PR DESCRIPTION
**Pull request by kushalbakshi**

### CODE REVIEW

File: element_calcium_imaging/imaging_no_curation.py [hunk 1]
```diff
@@ -1532,7 +1532,17 @@ def make(self, key):
...
-        caiman_params = (ProcessingTask * ProcessingParamSet & key).fetch1("params")
+        if "indicies" in params:
+            caiman_params = {
+                "motion": {
+                    "indicies": (
+                        slice(*params["indicies"][0]),
+                        slice(*params["indicies"][1]),
+                    )
+                }
+            }
```
Comment (ttngu207) [inline element_calcium_imaging/imaging_no_curation.py:1543]: I don't think you need to nest this inside `motion`

Comment (kushalbakshi) [reply element_calcium_imaging/imaging_no_curation.py:1543]: I did it this way based on my understanding of the documentation [here](https://caiman.readthedocs.io/en/master/Getting_Started.html) under the `Parameters` subheading. Let me know if you think it is still not necessary to nest the parameters like that

Comment (ttngu207) [reply element_calcium_imaging/imaging_no_curation.py:1543]: and this implementation would discard anything in `params`, replacing it with just the `motion` part

**Comment by kushalbakshi** [comment]
This is not needed given [element-interface #90](https://github.com/datajoint/element-interface/pull/90)

**Pull request closed without merging**